editor/gui/editor_file_dialog.cpp

```diff
@@ -766,6 +767,29 @@ void EditorFileDialog::update_file_name() {
 	}
 }
 
+// TODO: Could use a unit test.
+Color EditorFileDialog::get_dir_icon_color(const String &p_dir_path) {
+	// Creating these two variables every time might be unoptimized, although I don't think it's noticeable.
```

Comment:
These variables already exist in FileSystemDock, you are just fetching a reference, so it's cheap.
I think the comment is unnecessary.